api/src/domain/user/updateOperatingPhase.test.ts

```diff
@@ -114,22 +114,7 @@ describe("updateOperatingPhase", () => {
       expect(getCurrentBusiness(result).profileData.operatingPhase).toBe("NEEDS_TO_FORM");
     });
 
-    it("updates the phase to NEEDS_TO_REGISTER_FOR_TAXES from NEEDS_BUSINESS_STRUCTURE for TradeName", () => {
-      const userData = generateUserDataForBusiness(
-        generateBusiness({
-          profileData: generateProfileData({
-            businessPersona: "STARTING",
-            operatingPhase: "NEEDS_BUSINESS_STRUCTURE",
-            legalStructureId: "general-partnership",
-          }),
-          taskProgress: { [businessStructureTaskId]: "COMPLETED" },
-        })
-      );
-      const result = updateOperatingPhase(userData);
-      expect(getCurrentBusiness(result).profileData.operatingPhase).toBe("NEEDS_TO_REGISTER_FOR_TAXES");
-    });
-
-    it("updates status to FORMED_AND_REGISTERED when formation and business structure tasks are completed and PublicFiling legal structure and tax task is complete", () => {
+    it("updates status to FORMED when formation and business structure tasks are completed and PublicFiling legal structure", () => {
```

Comment:
Pebble - because of the OP change, the staged data and the titles are misaligned. Can you go through this file to make sure that the excess stage data is removed and the titles of reflect what changes are happening